src/commands/server/delete.ts

```diff
@@ -73,13 +73,21 @@ export default class Delete extends Command {
     const devWorkspaceTasks = new DevWorkspaceTasks(flags)
 
     const tasks = new Listrq([], ctx.listrOptions)
+    const checlusters = await kube.getAllCheClusters()
+
     tasks.add(apiTasks.testApiTasks(flags))
     tasks.add(operatorTasks.deleteTasks(flags))
-    tasks.add(olmTasks.deleteTasks(flags))
     tasks.add(cheTasks.deleteTasks(flags))
     tasks.add(helmTasks.deleteTasks(flags))
     tasks.add(cheTasks.waitPodsDeletedTasks())
 
+    // Remove 'openshift-operators' subscription in case if exists at least 1 checluster
```

Comment:
I don't like it.
I don't understand how subscription checking relates to all olmTasks